games/l.yaml

```
@@ -86,6 +86,30 @@
   type: remake
   updated: 2017-10-15
 
+- name: LBreakoutHD
+  originals:
+  - Breakout
+  type: clone
+  repo: 'http://prdownloads.sourceforge.net/lgames/lbreakouthd-1.0.5.tar.gz'
+  url: 'http://lgames.sourceforge.net/LBreakoutHD/'
+  development: sporadic
+  status: playable
+  content: free
```

```suggestion
  content: open
```
assuming they are also GPL2 licensed